ow/launch/common.launch

```diff
@@ -6,6 +6,8 @@
   <arg name="gazebo" default="true"/>
   <arg name="gzclient" default="true"/>  <!-- Only has effect if gazebo = true -->
   <arg name="record_bag" default="false"/>
+  <arg name="power_draw_csv_file" default="onewatt.csv"/>  <!-- options: onewatt.csv, eightwatt.csv, sixteenwatt.csv -->
+  <arg name="power_update_rate" default="10"/>  <!-- Use 10 Hz for onewatt.csv, 1 Hz for the two other options -->
```

Comment:
The 10 Hz was the incorrect one, not the 10s. onewatt.csv should be publishing at 10s intervals. I'll fix both here and in the readme.